graphdb/janus/src/main/java/org/apache/atlas/repository/graphdb/janus/AtlasElasticsearchQuery.java

```diff
@@ -234,6 +263,15 @@ private DirectIndexQueryResult performAsyncDirectIndexQuery(SearchParams searchP
             }
         }catch (Exception e) {
             LOG.error("Failed to execute direct query on ES {}", e.getMessage());
+
+            if (e instanceof SocketTimeoutException ||
+                    e instanceof UnknownHostException ||
+                    (e.getMessage() != null &&
+                            (e.getMessage().contains("Connection reset by peer") ||
+                                    e.getMessage().contains("Network error")))) {
+                throw new AtlasBaseException(AtlasErrorCode.SERVICE_UNAVAILABLE, e.getMessage());
```

Comment:
Please add the service name as well in the error message:
Service is unavailable or a network error occurred: Elasticsearch